index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <section id="welcome-section" class="welcome-section">
+        <h1>Hey am Jumite</h1>
+        <p>am a New web developer and designer</p>
+
+      </section>
+      <section id="projects">
+          <h2 class="projects-section-header">These are 
+              some of my projects
+          </h2>
```

Comment:
- [ ] Please ensure the content for the header tag is rightly indented, this will ensure your code is easily readable.

Reply:
Great work for implementing the stipulated guidelines on how to give a review with a 'why' a change has to be made